packages/shop-mobile-ionic/src/app/pages/+products/products.page.ts

```diff
@@ -155,13 +157,11 @@ export class ProductsPage implements OnInit, OnDestroy {
 	toggleGetProductsType() {
 		this.changePage = true;
 		this.products = [];
-		this.loadProducts({
-			count: this.lastLoadProductsCount,
-			imageOrientation: this.lastImageOrientation,
-		});
+		this.loadProducts();
```

Comment:
@sunk0 why you removed "count" and "imageOrientation" parameters? Do you understand why they were added here? What affect you making when you remove them? (hint: you broke a LOT)